tutorials/hess.ipynb

```diff
@@ -323,7 +324,42 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(analysis.fit.result[0])"
+    "print(model_pwl)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(10, 8))\n",
+    "crab_ref = CrabSpectrum(\"hess_pl\").model\n",
+    "\n",
+    "dataset_fp = analysis.spectrum_result\n",
+    "\n",
+    "plot_kwargs = {\n",
+    "    \"energy_range\": [1, 10] * u.TeV,\n",
+    "    \"flux_unit\": \"erg-1 cm-2 s-1\",\n",
+    "    \"energy_power\": 2,\n",
+    "}\n",
+    "\n",
+    "model_kwargs = {\"label\": \"1D best fit model\"}\n",
+    "model_kwargs.update(plot_kwargs)\n",
+    "ax_spectrum, ax_residuals = dataset_fp.peek(model_kwargs=model_kwargs)\n",
+    "\n",
+    "crab_ref.plot(ax=ax_spectrum, label=\"H.E.S.S. 2006 PWL\", **plot_kwargs)\n",
+    "model.spectral_model.plot(ax=ax_spectrum, label=\"3D best fit model\", **plot_kwargs)\n",
+    "\n",
+    "ax_spectrum.set_ylim(1e-11, 1e-10)\n",
+    "ax_spectrum.legend();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The difference we see compared to the reference H.E.S.S. 2006 model is because of the appilcation of the containment correction. "
```

Comment:
application

Reply:
The difference should not come from containment correction. It is also applied in the 2006 Crab paper.

Reply:
As a quick check I found that the un-corrected spectrum is much closer to the 2006 crab reference, but as I didn't check in detail it's probably better to remove the statement again.